Cargo.toml

```diff
@@ -33,7 +33,7 @@ enumflags2 = "0.7"
 futures-channel = "0.3"
 futures-util = "0.3"
 gdk4wayland = { package = "gdk4-wayland", version = "0.9", optional = true }
-gdk4x11 = { package = "gdk4-x11", version = "0.9", optional = true }
+gdk4x11 = { package = "gdk4-x11", version = "0.9", optional = true, features = [ "xlib" ] }
```

Comment:
this will enable the xlib feature all the time, whether you enable the backend feature or not... not sure what to do here

Reply:
I think it's fine as there won't be any breaking change in `xlib` and so we need not worry about breaking API contract?

Reply:
as the dependency itself is rather minimal, let us not block the situation forever.